app/code/AKisilenko/ModuleLesson11/view/frontend/templates/result_view.phtml

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @var \AKisilenko\ModuleLesson11\Block\View $block;
+ */
+?>
+    <div class="green" style="color: green;">
+        <h2>Show info from Di.xml</h2>
+        <?php
+        var_dump($block->getInjectObject());
+        ?>
+    </div>
+
+    <div class="red" style="color: indianred;">
+        <h2>File list </h2>
+        <?php
+        foreach ($block->getFileListObject()->makeFileList() as $item) {
```

Comment:
Please try colon variant of foreach (the same approach can be used with other control structures: if, else...
It is commonly used in Magento templates:
<?php foreach ($block->getFileListObject()->makeFileList() as $item): ?>
    <li> <?php echo $item; ?> <?php echo date('F d Y H:i:s.', filemtime($item)); ?>
<?php endforeach; ?>
!!! please check if it works, it's just an example.
Please refer Magento code examples https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Directory/view/frontend/templates/currency.phtml#L32

Reply:
done